transformer_engine/pytorch/module.py

```diff
@@ -454,10 +456,16 @@ def prepare_forward(
             self.set_fp8_weights()
 
             update_weight_scale_inv = is_first_microbatch is None or is_first_microbatch
+            reduce_amax = self.fp8_meta["recipe"].reduce_amax or self.sequence_parallel
+            if self.fp8 and self.sequence_parallel and not self.fp8_meta["recipe"].reduce_amax:
```

Comment:
It should be a one-time warning, let's not spam the user with this message every single forward pass.

Reply:
It will be a one time warning by default.

Reply:
Hmm, you're right. You learn something new every day :-).